client/src/app/site/pages/organization/pages/accounts/pages/account-list/components/account-list/account-list.component.html

```diff
@@ -63,7 +63,8 @@ <h2>{{ 'Accounts' | translate }}</h2>
                 </div>
                 <div class="meta-info user-subtitle ellipsis-overflow">
                     {{ user.saml_id || user.username }}
-                    <ng-container *ngIf="user.email">· {{ user.email }}</ng-container>
+                    <ng-container *ngIf="user.email">· {{ user.email }}&nbsp;</ng-container>
```

Comment:
```suggestion
                    <ng-container *ngIf="user.email">&middot; {{ user.email }}&nbsp;</ng-container>
```
Change this here too.

Reply:
done